pkgs/by-name/pd/pdfarranger/package.nix

```diff
@@ -6,6 +6,7 @@
   gtk3,
   poppler_gi,
   libhandy,
```

Comment:
please take stdenv, gettext as argument instead of pkgs. I think pkgs breaks splicing, which is important for cross compilation.